packages/seducer/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@paprika/seducer",
+  "version": "1.1.1",
+  "description": "Seducer (simple reducer) is a wrap on top of React.useReducer which make it easier to use it",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika.git",
+    "directory": "packages/seducer"
+  },
+  "keywords": [
+    "Context",
+    "React",
+    "dispatch",
+    "inversion of control",
+    "reducer",
+    "useReducer"
+  ],
+  "bugs": {
+    "url": "https://github.com/acl-services/paprika/issues"
+  },
+  "homepage": "https://github.com/acl-services/paprika/tree/master/packages/seducer",
+  "dependencies": {
+    "@paprika/button": "^1.0.3",
+    "@paprika/card": "^1.0.4",
+    "@paprika/collapsible-card": "^1.1.8",
+    "@paprika/toast": "^1.0.6",
```

Comment:
Unnecessary dependencies

Reply:
ohh those should be on dev dependencies, are used in the documentation :)